component/pom.xml

```diff
@@ -20,36 +20,36 @@
     <dependency>
       <groupId>com.celements</groupId>
       <artifactId>celements-subsystem-migration-manager</artifactId>
-      <version>3.0</version>
+      <version>4.0-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.celements</groupId>
       <artifactId>celements-model</artifactId>
-      <version>3.9</version>
+      <version>4.0-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.celements</groupId>
       <artifactId>celements-core</artifactId>
-      <version>3.6</version>
+      <version>4.0-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.celements</groupId>
       <artifactId>celements-search</artifactId>
-      <version>3.1</version>
+      <version>4.0-SNAPSHOT</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.drewnoakes</groupId>
       <artifactId>metadata-extractor</artifactId>
-      <version>2.6.2</version>
+      <version>2.7.2</version>
```

Comment:
why not upgrading to latest release 2.11.0 (see https://github.com/drewnoakes/metadata-extractor/releases )

Reply:
because of compile errors, I've picked the latest working release

Reply:
added https://synjira.atlassian.net/browse/CELDEV-816